sim/paladin/item_sets_pve.go

```diff
@@ -120,3 +120,63 @@ var ItemSetLawbringerRadiance = core.NewItemSet(core.ItemSet{
 		},
 	},
 })
+
+///////////////////////////////////////////////////////////////////////////
+//                            SoD Phase 5 Item Sets
+///////////////////////////////////////////////////////////////////////////
+
+var ItemSetFreethinkersArmor = core.NewItemSet(core.ItemSet{
+	Name: "Freethinker's Armor",
+	Bonuses: map[int32]core.ApplyEffect{
+		2: func(agent core.Agent) {
+			c := agent.GetCharacter()
+			c.AddStats(stats.Stats{
+				stats.HolyPower:       14,		
+			})
+		},
+		3: func(agent core.Agent) {
+			//Increases damage done by your holy shock spell by 50%
+			paladin := agent.GetCharacter()
+			paladin.OnSpellRegistered(func(spell *core.Spell) {
+				if spell.SpellCode == SpellCode_PaladinHolyShock {
+					spell.DamageMultiplier *= 1.5
+				}
+			})
+		},
+		5: func(agent core.Agent) {
+			// Reduce cooldown of Exorcism by 3 seconds
+			
```

Comment:
Looks like this implementation got missed?

Reply:
Huh that's weird, I'll pull down the branch and play around with it

Reply:
you need to refresh / update the aura

Reply:
I put up a PR against your branch https://github.com/mlizard32/wowsims_sod/pull/1! I updated the purifying power code a bit then changed the ZG 5-set to an aura with an `OnInit` to make sure it gets applied after the purifying power rune. This means that Exorcism with both will go from 15 / 2 = 7.5 - 3 = 4.5 seconds. I don't know for sure if the game will do that too or if it will do 15 - 3 = 12 / 2 = 6 but I feel like it'll be 4.5 seconds.

The issue was that you have to modify the exorcism spell's cooldown directly because `paladin.exorcismCD` is passed as a value. Really we should be storing the exorcism timer, as that's still a pointer

Reply:
Sounds good, merged that